fem/fespace.cpp

```diff
@@ -1277,7 +1278,7 @@ SparseMatrix* FiniteElementSpace::DerefinementMatrix(int old_ndofs,
    for (int k = 0; k < dtrans.embeddings.Size(); k++)
    {
       const Embedding &emb = dtrans.embeddings[k];
-      const Geometry::Type geom = mesh->GetElementBaseGeometry(emb.parent);
+      Geometry::Type geom = mesh->GetElementBaseGeometry(emb.parent);
```

Comment:
Why remove the `const`?

Reply:
I know it's a matter of taste but it seems superflous to mark every local variable that's not being written to by const, one ends up having const everywhere. I like to use const where necessary only (const pointers/refs, input parameters, const methods).